GitUI/UserControls/RevisionGrid.cs

```diff
@@ -3308,35 +3308,11 @@ public void GoToRef(string refName, bool showNoRevisionMsg)
             }
         }
 
-        /// <summary>
-        /// duplicated from GitExtensionsForm
-        /// </summary>
-        /// <param name="commandCode"></param>
-        /// <returns></returns>
-        private Keys GetShortcutKeys(int commandCode)
-        {
-            var hotkey = GetHotkeyCommand(commandCode);
-            return hotkey == null ? Keys.None : hotkey.KeyData;
-        }
-
         internal Keys GetShortcutKeys(Commands cmd)
         {
             return GetShortcutKeys((int)cmd);
         }
 
-        /// <summary>
```

Comment:
what about `internal Keys GetShortcutKeys(Commands cmd)`? both files have this method

Reply:
they look the same, but of different types, one is `RevisionDiff.Commands`, whereas the other one is `RevisionGrid.Commands`

Reply:
the cast the command down to int anyway